src/Graphics/Texture.cpp

```diff
@@ -306,4 +306,36 @@ bool Texture::blitWithAlpha(Texture* blitMask, int maskOffsetX, int maskOffsetY)
     return true;
 }
 
+// static
+std::unique_ptr<Texture> Texture::generateTextureForNumber(
+    unsigned int number,
+    unsigned int maxLength,
+    Texture* symbolSource,
+    unsigned int charWidth,
+    unsigned int charHeight,
+    unsigned int xOffsetByColor,
+    bool isSigned)
+{
+    // number as text
+    const auto charsCount = maxLength + (isSigned ? 1 : 0);
+    auto texture = std::unique_ptr<Texture>(new Texture(charWidth * charsCount, charHeight));
```

Comment:
Unfortunately not now (
It's c++14 featute and my STL library doesn't support it.